### PR TITLE
Update buildtools to version 01008-01

### PR DESCRIPTION
**Pull request by chcosta**
Fix for build issue on Alpine and Fedora23 builds

2016-11-13T08:40:57.7870990Z /root/coreclr/Tools/FrameworkTargeting.targets(67,3): error MSB4019: The imported project "/root/coreclr/Tools/Extensions/Microsoft/Portable/v4.5/Microsoft.Portable.CSharp.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [/root/coreclr/src/ToolBox/SOS/NETCore/SOS.NETCore.csproj]
2016-11-13T08:40:57.7878360Z /root/coreclr/Tools/FrameworkTargeting.targets(67,3): error MSB4019: The imported project "/root/coreclr/Tools/Extensions/Microsoft/Portable/v4.5/Microsoft.Portable.CSharp.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [/root/coreclr/src/mscorlib/facade/mscorlib.csproj]

/cc @gkhanna79 @ericstj

### CODE REVIEW

File: BuildToolsVersion.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0.27-prerelease-01007-02
\ No newline at end of file
+1.0.27-prerelease-01008-01
```
Comment (gkhanna79) [inline BuildToolsVersion.txt]: Were you able to validate that this fixes the issue?

**Comment by gkhanna79** [comment]
Modulo the comment, LGTM.

File: BuildToolsVersion.txt (hunk 1 above)
Comment (chcosta) [inline BuildToolsVersion.txt]: This is a known issue which was fixed in CoreFx.  I haven't done any explicit validation for this in CoreClr though.

**Comment by gkhanna79** [comment]
It will be good to validate to confirm that this fixes the issue.

File: BuildToolsVersion.txt (hunk 1 above)
Comment (chcosta) [inline BuildToolsVersion.txt]: I've validated on a Linux Vm running Fedora 23, that this fixes the issue.

**Pull request merged**